library/libwaku.nim

```diff
@@ -72,12 +72,22 @@ proc waku_new(configJson: cstring,
   if isNil(onErrCb):
     return RET_MISSING_CALLBACK
 
-  let createThRes = waku_thread.createWakuThread(configJson)
+  ## Create the Waku thread that will keep waiting for req from the main thread.
+  let createThRes = waku_thread.createWakuThread()
   if createThRes.isErr():
     let msg = "Error in createWakuThread: " & $createThRes.error
     onErrCb(unsafeAddr msg[0], cast[csize_t](len(msg)))
     return RET_ERR
```

Comment:
```suggestion
  let createTh = waku_thread.createWakuThread().valueOr:
    let msg = "Error in createWakuThread: " & $error
    onErrCb(unsafeAddr msg[0], cast[csize_t](len(msg)))
    return RET_ERR
```

Reply:
or.. if this is `Result[void]`, use `isOkOr:`

Reply:
Super interesting! Thanks for that! Notice that I will only apply that to this snippet and I'm planning to extend this approach to further places at `nwaku`.